File: critreel/critreel_codigo/critreel_ionic/projeto/src/app/model/usuario.ts
export class Usuario {
    idUsuario: number;
    username: string;
    senha: string;
    dtNasc: string;
    suspensao: boolean;
    nomeCompleto: string;
    bioPerfil: string;
    fotoPerfil: string;
    email: string;
    admin: boolean;

    constructor() {
        this.idUsuario = 0;
        this.username = '';
        this.senha = '';
        this.dtNasc = '';
        this.suspensao = false;
        this.nomeCompleto = '';
        this.bioPerfil = '';
        this.fotoPerfil = '';
        this.email = '';
        this.admin = false;
    }
}

File: critreel/critreel_codigo/critreel_api/critreelFinal/critreel/src/main/java/br/cefetmg/critreel/model/Usuario.java
package br.cefetmg.critreel.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Usuario {
    private Long idUsuario;
    private String username;
    private String senha;
    private String dtNasc;
    private boolean suspensao;
    private String nomeCompleto;
    private String bioPerfil;
    private String fotoPerfil;
    private String email;
    private boolean admin; // indica se o usuário é administrador
}
